### PR TITLE
[9.0][MIG] account_due_list_payment_mode: Migrate to V9

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @tecnativa

### CODE REVIEW

File: account_due_list_payment_mode/__init__.py [hunk 1]
```diff
@@ -1,24 +1,3 @@
 # -*- encoding: utf-8 -*-
```
Comment (pedrobaeza) [inline account_due_list_payment_mode/__init__.py]: coding

File: account_due_list_payment_mode/__openerp__.py [hunk 2]
```diff
@@ -1,41 +1,20 @@
 # -*- encoding: utf-8 -*-
```
Comment (pedrobaeza) [inline account_due_list_payment_mode/__openerp__.py]: coding

File: account_due_list_payment_mode/models/__init__.py [hunk 3]
```diff
@@ -1,24 +1,2 @@
 # -*- encoding: utf-8 -*-
```
Comment (pedrobaeza) [inline account_due_list_payment_mode/models/__init__.py]: coding

File: account_due_list_payment_mode/views/payment_view.xml [hunk 4]
```diff
@@ -1,31 +1,31 @@
...
+    <record id="view_payments_mode_filter" model="ir.ui.view">
+        <field name="name">Payments Mode Select</field>
+        <field name="model">account.move.line</field>
+        <field name="inherit_id" ref="account_due_list.view_payments_filter"/>
+        <field name="arch" type="xml">
+            <field name="date_maturity" position="after">
+                <field name="payment_mode_id"/>
             </field>
-        </record>
+            <filter string="Month" icon="terp-go-month" domain="[]"
```
Comment (pedrobaeza) [inline account_due_list_payment_mode/views/payment_view.xml]: Put only `<filter context="{'group_by':'date_maturity'}"` as selector

File: account_due_list_payment_mode/views/payment_view.xml [hunk 5]
```diff
@@ -1,31 +1,31 @@
...
-            <field name="inherit_id" ref="account_due_list.view_payments_filter"/>
-            <field name="arch" type="xml">
-                <field name="date_maturity" position="after">
-                    <field name="payment_mode_id"/>
-                </field>
-                <filter string="Month" icon="terp-go-month" domain="[]" context="{'group_by':'date_maturity'}" position="after">
-                    <filter string="Payment Mode" icon="terp-dolar" domain="[]" context="{'group_by':'payment_mode_id'}"/>
-                </filter>
+    <record id="view_payments_mode_filter" model="ir.ui.view">
+        <field name="name">Payments Mode Select</field>
```
Comment (pedrobaeza) [inline account_due_list_payment_mode/views/payment_view.xml]: Don't put name on inherited views.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
@sergio-teruel please make the requested changes

File: account_due_list_payment_mode/views/payment_view.xml (hunk 5 above)
Comment (mileo) [reply account_due_list_payment_mode/views/payment_view.xml]: @pedrobaeza You are sayng to don't use an explict name just a namespace like:

https://github.com/OCA/account-analytic/blob/10.0/account_analytic_no_lines/views/account_analytic_account.xml#L8

```
<record model="ir.ui.view" id="account_analytic_account_form_view">
        <field name="name">account.analytic.account.form (in account_analytic_no_lines)</field>
        <field name="model">account.analytic.account</field>
        <field name="inherit_id" ref="analytic.view_account_analytic_account_form"/>
        <field name="arch" type="xml">
            <xpath expr="//div[@name='button_box']/button[@name=%(analytic.account_analytic_line_action)d]" position="attributes">
                <attribute name="groups">account_analytic_no_lines.hide_analytic_entries</attribute>
            </xpath>
        </field>
    </record>
```

In this case
```
<field name="name">account.move.select (in account_due_list_payment_mode)</field>
```

cc: @vonpupp

**Comment by mileo** [comment]
LGTM, after correcting Baeza's comments.

File: account_due_list_payment_mode/views/payment_view.xml (hunk 5 above)
Comment (pedrobaeza) [reply account_due_list_payment_mode/views/payment_view.xml]: I say that for avoiding extra translation and lines, as this name is useless and it's auto-generated if not explicitly added, but you can put it if you want.

**Review by pedrobaeza — APPROVED** [review]

**Comment by LoisRForgeFlow** [comment]
Runbot is having issues with the account_due_list dependency, does this need a rebase?

**Comment by pedrobaeza** [comment]
It seems so. Rebased to see the result.

**Comment by vonpupp** [comment]
PR #129 seems to address that issue. I will try it later.

**Comment by vonpupp** [comment]
I have created a simple [buildout] for testing purposes. Please use the branch `9-mig-account_due_list_payment_mode`. I tested the installation and it seems ok.

[buildout]: https://github.com/vonpupp/testpr-buildout

**Review by luismontalba — APPROVED** [review]
Tested on runbot

**Review by mileo — APPROVED** [review]

**Comment by mileo** [comment]
Tested on runbot

**Comment by vonpupp** [comment]
I believe this is ready for merge @pedrobaeza, isn't it?

**Pull request merged**